server/scripts/cli.py

```diff
@@ -0,0 +1,86 @@
+import csv
+import json
+from typing import BinaryIO, Dict
+
+import click
+
+from dive_server.serializers import kwcoco, viame
+from dive_utils import models
+from scripts import generateLargeDataset
```

Comment:
`generateLargeDataset` imports `numpy`, `opencv`, and `tqdm` and since `cli.py` imports `generateLargeDataset`, it also require those imports. This isn't an issue if the dev requirements are installed, but if the user wants to use the CLI, they would need the dev requirements.